### PR TITLE
Add buttons for the new ui elements

### PR DESCRIPTION
**Pull request by priyesh777**
- Add different types of button as a part of UI elements

### CODE REVIEW

File: src/root/views/GoUI/styles.module.scss [hunk 1]
```diff
@@ -1,154 +1,186 @@
...
+  /* 4px */
+  --go-ui-border-radius-lg: 0.5rem;
+  /* 8px */
+  --go-ui-border-radius-xl: 0.75rem;
+  /* 12px */
+  --go-ui-border-radius-2xl: 1rem;
+  /* 16px */
+  --go-ui-border-radius-3xl: 1.5rem;
+  /* 24px */
+  --go-ui-border-radius-full: 9999px;
```
Comment (samshara) [inline src/root/views/GoUI/styles.module.scss:95]: Let's put the comments to side of the variables as before.

File: src/root/views/GoUI/styles.module.scss [hunk 2]
```diff
@@ -1,154 +1,186 @@
...
+  --go-ui-color-blue-70: #4D617A;
+  --go-ui-color-blue-80: #344B67;
+  --go-ui-color-blue-90: #011E41;
+
+  --go-ui-color-positive: var(--go-ui-color-green);
+  --go-ui-color-information: var(--go-ui-color-blue);
+  --go-ui-color-warning: var(--go-ui-color-yellow);
+  --go-ui-color-negative: var(--go-ui-color-red);
+
+  --go-ui-color-background: #F7F7F7;
```
Comment (samshara) [inline src/root/views/GoUI/styles.module.scss:173]: Let's remove this definition as it is already defined in line 156.

File: src/root/views/GoUI/hooks/index.tsx
[the remark this answers is not in the document]
Comment (priyesh777) [reply src/root/views/GoUI/hooks/index.tsx:1]: This file will be used for drop-down after this branch will be merged.

**Review by shreeyash07 — CHANGES_REQUESTED** [review]

**Review by shreeyash07 — APPROVED** [review]

File: src/root/components/Heading/index.tsx
Comment (samshara) [inline src/root/components/Heading/index.tsx:1]: Remove this unused component

File: src/root/views/GoUI/hooks/index.tsx
Comment (samshara) [inline src/root/views/GoUI/hooks/index.tsx:1]: Let's remove this.

File: src/root/components/Heading/index.tsx
Comment (samshara) [inline src/root/components/Heading/index.tsx:1]: Let's remove this.

**Review by samshara — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**